### PR TITLE
HDDS-3100. Fix TestDeadNodeHandler.

### PR DESCRIPTION
**Pull request by avijayanhwx**
## What changes were proposed in this pull request?

Fix issue in TestNodeHandler which was caused due to the race condition between the 3 Factor pipeline creation in SCM, and the explicit opening of all pipelines in the Test.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-3100

## How was this patch tested?
Ran the test a few times in the IDE and command line.

### CODE REVIEW

**Comment by avijayanhwx** [comment]
Unit test failure (TestBlockManager) is unrelated.

**Comment by adoroszlai** [comment]
> Unit test failure (TestBlockManager) is unrelated.

Being fixed in HDDS-2989.

File: hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestDeadNodeHandler.java [hunk 1]
```diff
@@ -89,16 +89,16 @@ public void setup() throws IOException, AuthenticationException {
     storageDir = GenericTestUtils.getTempPath(
         TestDeadNodeHandler.class.getSimpleName() + UUID.randomUUID());
     conf.set(HddsConfigKeys.OZONE_METADATA_DIRS, storageDir);
-    conf.setInt(OZONE_DATANODE_PIPELINE_LIMIT, 0);
     eventQueue = new EventQueue();
+    System.out.println("eventQueue = "  + eventQueue.toString());
```
Comment (adoroszlai) [inline hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestDeadNodeHandler.java]: Leftover debug.

**Comment by avijayanhwx** [comment]
Thank you for the review @adoroszlai.

**Comment by bharatviswa504** [comment]
Thank You @avijayanhwx for the contribution and @adoroszlai for the review.

**Pull request merged**